idf_build_apps/app.py

```diff
@@ -533,8 +533,10 @@ def build(
             for line in lines:
                 is_error_or_warning, ignored = self.is_error_or_warning(line)
                 if is_error_or_warning:
-                    LOGGER.warning('%s', line)
-                    if not ignored:
+                    if ignored:
+                        LOGGER.info('>>> %s [Ignored warning]', line)
+                    else:
+                        LOGGER.warning('>>> %s', line)
```

Comment:
```suggestion
                        LOGGER.warning('%s', line)
```

the `>>> ` is added automatcially for warnings https://github.com/espressif/idf-build-apps/blob/main/idf_build_apps/log.py#L41